.packit.yaml

```diff
@@ -31,10 +31,10 @@ jobs:
   - job: copr_build
     trigger: pull_request
     targets:
-      centos-stream-8:
+      rhel-8:
         additional_modules: "foreman:el8"
         additional_repos:
-          - http://koji.katello.org/releases/yum/foreman-nightly/el8/x86_64/
+          - https://stagingyum.theforeman.org/foreman/3.9/el8/x86_64/
```

Comment:
in other places (also here in `develop`), we went ahead and started to use `yum.tfm.o` directly